spacy/tests/doc/test_span.py

```diff
@@ -98,6 +98,11 @@ def test_spans_span_sent(doc, doc_not_parsed):
     assert doc[:2].sent.root.text == "is"
     assert doc[:2].sent.text == "This is a sentence."
     assert doc[6:7].sent.root.left_edge.text == "This"
+    assert doc[0: len(doc)].sent == list(doc.sents)[0]
+
+    with pytest.raises(ValueError) :
+        doc_not_parsed[:2].sent
```

Comment:
I think we can remove this part from this test, as it's already included in a later test (where it's using `span = Span(doc_not_parsed, 0, 3)`)

Reply:
This is a valid test for non-sentecized doc. In the following lines, the sentence boundaries are set manually and the assertions pass without throwing an exception:
```python
   with pytest.raises(ValueError):
        doc_not_parsed[:2].sent

    # test on manual sbd
    doc_not_parsed[0].is_sent_start = True
    doc_not_parsed[5].is_sent_start = True
    assert doc_not_parsed[1:3].sent == doc_not_parsed[0:5]
    assert doc_not_parsed[10:14].sent == doc_not_parsed[5:]

```

Reply:
I'm just trying to be mindful of not repeating the same kind of test too often ;-)